docs/topics/multiplatform/multiplatform-android-layout.md

```diff
@@ -0,0 +1,108 @@
+[//]: # (title: Android source set layout)
+
+The new Android source set layout was introduced in Kotlin 1.8.0 and became default in 1.9.0. Follow our guide to
+understand the key differences between the deprecated and the new layout and migrate your projects.
+
+> You don't need to implement all the suggestions. Only those that are applicable to your particular projects. 
+> 
+{type="tip"}
+
+## Check the compatibility
+
+Check your version of the Android Gradle plugin and upgrade if necessary.
+The new layout requires Android Gradle plugin 7.0 or later and is supported in Android Studio 2022.3 and later.
+
+## Rename Kotlin source sets
+
+If applicable, rename source sets in your project, following this pattern:
+
+| Previous source set layout             | New source set layout               |
+|----------------------------------------|-------------------------------------|
+| `targetName` + `AndroidSourceSet.name` | `targetName` + `AndroidVariantType` |
+
+`{AndroidSourceSet.name}` maps to `{KotlinSourceSet.name}` as follows:
+
+|             | Previous source set layout | New source set layout          |
+|-------------|----------------------------|--------------------------------|
+| main        | androidMain                | androidMain                    |
+| test        | androidTest                | android<b>Unit</b>Test         |
+| androidTest | android<b>Android</b>Test  | android<b>Instrumented</b>Test |
+
+## Move source files
+
+If applicable, move your source files to the new directories, following this pattern:
+
+| Previous source set layout                            | New source set layout               |
+|-------------------------------------------------------|-------------------------------------|
+| The layout had additional `/kotlin` SourceDirectories | `src/{KotlinSourceSet.name}/kotlin` |
+
+`{AndroidSourceSet.name}` maps to `{SourceDirectories included}` as follows:
+
+|             | Previous source set layout                                 | New source set layout                                                                          |
+|-------------|------------------------------------------------------------|------------------------------------------------------------------------------------------------|
+| main        | src/androidMain/kotlin, src/main/kotlin, src/main/java     | src/androidMain/kotlin, src/main/kotlin, src/main/java                                         |
```

Comment:
Is it technically possible to have a new line for each layout name?
Something like `<br/>` or similar?

It will be easier to read and compare the layouts